### ISSUE TITLE
Navigation is empty on homepage

### ISSUE THREAD

**Issue by tomkerkhove** [cncf/tag-app-delivery#399]
Navigation is empty on homepage which makes it hard to find the whitepapers (and maybe other content as well?):

![image](https://user-images.githubusercontent.com/4345663/235593401-07b1a499-6fc1-4113-ab76-76fe7311db2c.png)

But when going to https://tag-app-delivery.cncf.io/whitepapers/platforms/, it shows them:

![image](https://user-images.githubusercontent.com/4345663/235593495-30e6c15f-dbfc-406b-9007-b6045c95f270.png)

**Comment by joshgav** [comment]
@lianmakesthings could you take a look at this? Could you see about adding the navigation items to the home page too? Thanks!

**Comment by joshgav** [comment]
@cjyabraham is it possible to list the _sections_ on the side of the homepage? Thanks!

**Comment by cjyabraham** [comment]
Yes, it should be possible by modifying the algorithm that manages that sidebar nav.

**Comment by cjyabraham** [comment]
I'm not sure exactly how you want this to work, however, I've started [a PR here](https://github.com/cncf/tag-app-delivery/pull/483). [Here is the dev instance](https://deploy-preview-483--tag-app-delivery.netlify.app/).

**Comment by joshgav** [comment]
Yes @cjyabraham that's great! Let's merge!
